src/main/java/com/ericsson/ei/controller/QueryAggregatedObjectControllerImpl.java

```diff
@@ -51,6 +51,6 @@
         ArrayList<String> response = processAggregatedObject.processQueryAggregatedObject(id);
         queryResponse.setResponseEntity(response.toString());
         LOGGER.debug("The response is : " + response.toString());
-        return new ResponseEntity(queryResponse, HttpStatus.OK);
+        return new ResponseEntity<>(queryResponse, HttpStatus.OK);
```

Comment:
Don't need to specify a type in between "< >" characters?

Reply:
no need. it will automatically take return type of this method